### DISCUSSION TITLE
[idea] Graph Mining: Laws, Tools, and Case Studies 教材改编

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#102] [3️⃣ 聊学术]
书籍：**Graph Mining: Laws, Tools, and Case Studies** (Deepayan Chakrabarti, Christos Faloutsos)
- 链接: https://www.morganclaypool.com/doi/abs/10.2200/S00449ED1V01Y201209DMK006
- 全文：https://xlab2017.yuque.com/msdpvs/fzem1h/26449587

这本图数据挖掘的教材，篇幅也不多，正文一百多页；可以基于这个材料做改编，即把里面所有的数据集和案例，全部换成开源场景下的数据和案例，就可以作为我们研究生的标准学习教材了，比较适合高校来做~

里面都是图数据分析中非常基础的数学工具和方法，在上面可以不断增加各种模块，例如可视化分析等~

里面不少内容加工下，可以直接用来发小论文～

**Comment by bifenglin** [comment]
比较系统，看了这本书目录，补全了图数据挖掘知识的短板。
